.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: python
+
+# This (sudo: false) is needed to "run on container-based infrastructure" on
+# which cache: is available
+# http://docs.travis-ci.com/user/workers/container-based-infrastructure/
+sudo: false
+
+cache: pip
+
+env: PIP_ALLOW_EXTERNAL=PIL PIP_ALLOW_UNVERIFIED=PIL
+
+before_script:
+  - curl -O http://geolite.maxmind.com/download/geoip/database/GeoLiteCountry/GeoIP.dat.gz
+  - curl -O http://geolite.maxmind.com/download/geoip/database/GeoLiteCity.dat.gz
+  - gzip -d GeoIP.dat.gz
+  - gzip -d GeoLiteCity.dat.gz
+  - mv Geo* omero_qa
+
+script: python manage.py test registry --settings=omero_qa.settings-test
```

Comment:
@aleksandra-tarkowska: is there a way to pass a verbose flag to list passed/failed tests in Travis?